src/io/pithos/util.clj

```diff
@@ -83,12 +84,74 @@
   []
   (javax.xml.bind.DatatypeConverter/printDateTime (Calendar/getInstance @utc)))
 
+
+(defn ^{:private true} formatter-with-zone
```

Comment:
private def and defn's make testing harder for little added value, I would be in favor of switching to plain def's